sbt-scalajs-bundler/src/sbt-test/sbt-scalajs-bundler/sbt-1.0.x/build.sbt

```diff
@@ -1,6 +0,0 @@
-enablePlugins(ScalaJSBundlerPlugin, ScalaJSJUnitPlugin)
-scalaVersion := "2.12.3"
-scalaJSUseMainModuleInitializer := true
-npmDependencies in Compile += "left-pad" -> "1.1.3"
-
-ivyLoggingLevel in ThisBuild := UpdateLogging.Quiet
```

Comment:
Shouldn’t we keep this test?

Reply:
It doesn't seem to test anything that isn't tested several times over in the other tests, so I thought it might as well be removed.

Reply:
OK, I didn’t double check, thanks!